### DISCUSSION TITLE
site not opening...

### DISCUSSION THREAD

**Discussion by SomilKotnala** [ibm-skills-network/author-workbench-knowledgebase#40] [Help]
I am studying from COURSERA, whenever I try to open a LAB via "Launch App"...the site displays an error. Help Please!!!!!!!!!!!!!

**Comment by michaelroudnitski** [comment]
Hi, sorry to hear you are seeing errors when trying to launch your lab. Can you please provide us with the following information so we can try to fix the problem for you?

- The course you are enrolled in
- The lab you tried to launch within the course

A link would be best, thank you.
